WooCommerce/WooCommerceTests/ViewRelated/OrderTrackingTableViewCellTests.swift

```diff
@@ -0,0 +1,105 @@
+
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class OrderTrackingTableViewCellTests: XCTestCase {
```

Comment:
This is interesting! I've never tested UI classes.

Reply:
This is great @ctarda — we should make a habit of doing this kind of test going forward! +1